syft/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Copyright © 2022 Booz Allen Hamilton. All Rights Reserved.
+# This software package is licensed under the Booz Allen Public License. The license can be found in the License file or at http://boozallen.github.io/licenses/bapl
+
+ARG BASE_REGISTRY=registry.access.redhat.com
+ARG BASE_IMAGE=ubi8/ubi
+ARG BASE_TAG=8.4
+FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
+
+### Required Atomic/OpenShift Labels - https://github.com/projectatomic/ContainerApplicationGenericLabels
+LABEL name="Solutions Delivery Platform: Syft" \
+      maintainer="terrana_steven@bah.com" \
```

Comment:
similar to #40, suggest updating to use your email, or a UIP mailbox

Reply:
changed to uip distro